tests/org.eclipse.reddeer.eclipse.test/src/org/eclipse/reddeer/eclipse/test/wst/server/ui/view/ServersViewTestCase.java

```diff
@@ -75,14 +77,18 @@ protected static void createServer(String name) {
 		new WaitUntil(new ServerExists(name));	
 	}
 	
-	protected static void importProjects(){
-		ExternalProjectImportWizardDialog wizard  = new ExternalProjectImportWizardDialog();
+	protected static void importProjects(boolean importMultimodular) {
```

Comment:
You can write something like:
importProjects(){
importProjects(false);
}
so you dont have to rewrite it on so many places

Reply:
You are right, this is much simpler and cleaner solution.